README.md

```diff
@@ -5,6 +5,8 @@ If you're cheap like me, you want good free stuff. Here's a list of all my favor
 ### Image Editing
 Bitmap/Raster Graphics: [Paint.NET](http://www.getpaint.net/index.html), [Gimp](http://www.gimp.org/)
 
+Pixel art: [Aseprite](http://www.aseprite.org/) *(Pre-compiled binaries cost, but you can download the source and compile for free)*
```

Comment:
I know this is kind of off-topic, but why would you sell the pre-compiled binaries but give out the source code for free? "Here, lazy people, give us $3177.99 and we'll give you an easy way to get started!"